docs/UserGuide.md

```diff
@@ -84,30 +81,23 @@
 
       Event Trick or treat starts today 8pm and ends on 31 Oct 9pm.
 
-   The events for current selected day will show up under schedule.
+   The events under schedule.
 
-   ![alt text](./images/userguide/toggle_calender.png "")
+10. All task in the database will be assigned a unique index. The unique index can be seen to the left of the task name.
 
-9. To see the schedule of a particular day, enter `view DATE`.
-   The schedule for the day will show up in the schedule column.
-
-	`view 10oct2016`
-
-   ![alt text](./images/userguide/event.png "")
-
-   To view all events, enter `view events`
-
-   ![alt text](./images/userguide/event_list.png "")
-
-10. Let's view the index of task. Enter `i`.
-
-    ![alttext](./images/userguide/index.png "")
-
-    Enter anything into command line to hide index view.
+    ![alttext](./images/userguide/index.png "")to update with actual app shots
```

Comment:
still i need to redo all the images or it wont make sense, i can get it done by latest 1120